protocol/invocation.go

```diff
@@ -34,15 +34,16 @@ type Invocation interface {
 	// Reply gets response of request
 	Reply() interface{}
 	// Attachments gets all attachments
-	Attachments() map[string]string
-	// AttachmentsByKey gets attachment by key , if nil then return default value
+	Attachments() map[string]interface{}
+	// AttachmentsByKey gets attachment by key , if nil then return default value. （It will be discarded）
```

Comment:
change it to 'deprecated in the future'?

Reply:
done